pkgs/top-level/all-packages.nix

```diff
@@ -2464,6 +2464,36 @@ in
   conspy = callPackage ../os-specific/linux/conspy {};
 
   connman = callPackage ../tools/networking/connman { };
+  connmanThin = callPackage ../tools/networking/connman {
```

Comment:
`connmanMinimal` is more idiomatic.

These long definitions belong to a separate file. Use the definition of `tesseract` as a template:
```
  inherit (callPackage ../tools/networking/connman {})
    connman
    connmanMinimal
    connmanFull;
```

Reply:
I'm not sure I understand the idea of this syntax.. Where should I put all of the `enableThis = true/false;` flags?

Reply:
Got it.